build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     compile 'org.apache.kafka:connect-api:0.9.0.0'
     compile 'org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.0.2'
     compile 'org.slf4j:slf4j-api:1.7.14'
+    compile 'org.slf4j:slf4j-simple:1.7.14'
```

Comment:
usuń, zrób clean'a i zobacz testy za pomocą Intellij'a.

`SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".`
`SLF4J: Defaulting to no-operation (NOP) logger implementation`
`SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.`

Nie umiem inaczej tego obejść.